app/uk/gov/hmrc/gform/formtemplate/FormTemplateValidator.scala

```diff
@@ -128,6 +128,23 @@ object FormTemplateValidator {
     }
   }
 
+  def validateEnrolmentSection(formTemplate: FormTemplate): ValidationResult =
+    formTemplate.authConfig match {
+      case HasEnrolmentSection(_, enrolmentSection) =>
+        val fcIds = enrolmentSection.fields.map(_.id).map(_.value).toSet
+        val ctxs = enrolmentSection.identifiers.map(_.value.value).toList.toSet ++ enrolmentSection.verifiers
+          .map(_.value.value)
+          .toSet
+        if (ctxs.subsetOf(fcIds))
```

Comment:
I don't know whether you like this kind of thing, but I added ValidationResult.BooleanToValidationResultSyntax to allow this kind of thing to be written as: ctxs.subsetOf(fcIds).validationResult(message)